frontend/src/interfaces/user.ts

```diff
@@ -1,26 +0,0 @@
-export const AccountTypes = ['User', 'Service Account'] as const;
-type AccountType = (typeof AccountTypes)[number];
-
-export interface IUser {
```

Comment:
This can be replaced with the server type from `@server/types`

Reply:
This didn't work well because the frontend interface has other fields and it was not easy to fix